go-controller/pkg/ovn/egressgw_test.go

```diff
@@ -66,8 +68,14 @@ var _ = ginkgo.Describe("OVN Egress Gateway Operations", func() {
 					namespaceT.Name,
 				)
 
-				t.baseCmds(fExec)
-				fakeOvn.start(ctx,
+				dbSetup := libovsdbtest.TestSetup{
```

Comment:
@JacobTanenbaum maybe this could be a small function called something like `defaultTestSetup()`, since almost every test seems to use it?

Reply:
I moved the database creation to `BeforeEach()` is that enough? or should I try and simplify it more?

Reply:
Nah, this looks good. Thanks.